src/hooks/useSnackbarContext.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { SnackbarContext } from '@/contexts/Snackbar'
+import { SnackbarContextValue } from '@/interfaces/snackbarContextValue'
+
+export const useSnackbarContext = (): SnackbarContextValue => {
```

Comment:
lets just write `useSnackbar` for this hook, same for the file name

Reply:
Fixed